packages/web/src/components/pages/TestSuites/TestSuitesList/TestSuitesList.tsx

```diff
@@ -58,50 +61,36 @@ const TestSuitesList: FC = () => {
     dataTestModalRoot: 'add-a-new-test-suite-modal',
   });
 
-  const [abortAll] = useAbortAllTestSuiteExecutionsMutation();
   const onItemClick = useDashboardNavigate((item: TestSuite) => `/test-suites/${item.name}`);
-  const onItemAbort = useCallback((item: TestSuite) => {
-    abortAll({id: item.name})
-      .unwrap()
-      .catch(() => {
-        notificationCall('failed', 'Something went wrong during test suite execution abortion');
-      });
-  }, []);
 
   if (error) {
     return <Error title={(error as any)?.data?.title} description={(error as any)?.data?.detail} />;
   }
 
-  if (entityPromoComponent) {
-    return (
-      <>
-        {isValidElement(entityPromoComponent)
-          ? cloneElement(entityPromoComponent, {list: 'test-suites'} as Partial<unknown>)
-          : null}
-      </>
-    );
+  if (EntityPromoComponent) {
+    return <EntityPromoComponent list="test-suites" />;
   }
 
   return (
-    <EntityListContent
+    <EntityView
       itemKey="testSuite.name"
       CardComponent={TestSuiteCard}
       onItemClick={onItemClick}
-      onItemAbort={onItemAbort}
       entity="test-suites"
       pageTitle="Test Suites"
       pageTitleAddon={pageTitleAddon}
       addEntityButtonText="Add a new test suite"
-      pageDescription={PageDescription}
+      pageDescription={<PageDescription />}
       emptyDataComponent={EmptyTestSuites}
       initialFiltersState={initialFilters}
       dataTest="add-a-new-test-suite-btn"
       queryFilters={filters}
       setQueryFilters={setFilters}
-      data={testSuites}
-      isLoading={isLoading}
+      data={testSuites ?? []}
```

Comment:
Better to allowed undefined - the `[]` passed here will always have a new identity, so it will always call `useEffect` that has it as a dependency:

```suggestion
      data={testSuites}
```